### DISCUSSION TITLE
[Question] How to enable Local Playback in Browser?

### DISCUSSION THREAD

**Discussion by Silipwn** [jcorporation/myMPD#339] [Q&A] [answered]
I followed the instructions in the [wiki](https://github.com/jcorporation/myMPD/wiki/Local-playback). But I can't listen to the music, in the browser itself. 
I can listen to the stream if I open a media player to connect to the stream URL directly. Is there something else that needs to be configured? 
I am serving everything in HTTP.

myMPD version: v6.7.0
mpd version: v0.21.20

**Comment by jcorporation** [comment, marked as answer]
If you have enabled local playback in the settings dialog (General tab) you should have a icon aside from the volume button.

**Comment by Silipwn** [comment]
Yeah that clears things out. Thanks for the help.
